Sources/KituraContracts/Contracts.swift

```diff
@@ -22,7 +22,7 @@
  and server side (eg Kitura) of the request (typically a HTTP REST request).
 
  ### Usage Example: ###
- ````
```

Comment:
On lines 21 and 22 we need full stops within the eg for both - e.g. KituraKit and e.g. Kitura

Reply:
Please can you move the text on lines 32 and 33 to be immediately above the associated code sample.
Also please add a comma after "In this example" so it reads "In this example, the `RequestError`..."